bigquery/indexer.py

```diff
@@ -180,6 +180,35 @@ def _docs_by_id_from_export(storage_client, bucket_name, export_obj_prefix,
         yield participant_id, row
 
 
+def _create_table_from_view(bq_client, view):
+    # Creates a table named {}_copy that is a copy of view into
+    # dataset 'dataset_for_view_exports'.
+    # Creates the dataset if it doesn't exist.
+    # Both the table and the dataset are created in the deploy project
+    dataset_ref = bq_client.dataset('dataset_for_view_exports')
+    try:
+        bq_client.get_dataset(dataset_ref)
+    except exceptions.NotFound:
+        dataset = bigquery.Dataset(dataset_ref)
+        dataset = bq_client.create_dataset(dataset)
+        logger.info('Created new dataset {}'.format(dataset.dataset_id))
+    new_table_name = '{}_copy'.format(view.table_id)
+    new_table_ref = dataset_ref.table(new_table_name)
+    new_table_job_config = bigquery.QueryJobConfig()
+    new_table_job_config.destination = new_table_ref
+    new_table_job_config.use_legacy_sql = view.view_use_legacy_sql
```

Comment:
Thanks, fixed

Reply:
It might be nice, however, to move over to using `format`. There's a note [here](https://docs.python.org/3/library/stdtypes.html?highlight=sprintf#printf-style-string-formatting) that explains some advantages.

Reply:
Can you point to exactly where it talks about the advantages? I couldn't find it.

Reply:
Ah thanks. Sounds good, we can use .format() for new code.